templates/views/ControllerInputsView.html

```diff
@@ -0,0 +1,13 @@
+<div class="square back onefour">
+  <div class="circle front onefour">
+    <p class="value one">0,</p><p class="value four">0</p>
+  </div>
+</div>
+
+<div class="square back twothree">
+  <div class="circle front twothree">
```

Comment:
you should't need to declare a child with the same class as a parent, as it defeats the purpose of having a class. You'll have to modify your css to account for this